dashboard/test/models/user_test.rb

```diff
@@ -892,8 +892,9 @@ def assert_fails_email_uniqueness_validation(user)
         best_result: Activity::MINIMUM_PASS_RESULT
       )
     end
-    assert twenty_hour.script_levels.first.level.unplugged?
-    assert_equal(2, user.next_unpassed_visible_progression_level(twenty_hour).chapter)
+    user.stubs(:completed?).returns(true)
```

Comment:
Do we even need to create `UserLevels` above, if we're going to stub `completed?`

Or - would it be a more "realistic" test to do this?
```suggestion
    create :user_script, user: user, script: twenty_hour, completed_at: Time.now
```